From: Christoph Hellwig <hch@lst.de>
Date: Mon, 16 Aug 2021 14:45:29 +0200
Subject: nvme: remove the unused NVME_NS_* enum
Patch-mainline: v5.15-rc1
Git-commit: 9891668e43c8e9f2d0d50088b151edefc2e560e5
References: jsc#PED-1183

These values are unused now that the lightnvm support is gone.

Signed-off-by: Christoph Hellwig <hch@lst.de>
Reviewed-by: Keith Busch <kbusch@kernel.org>
Acked-by: Daniel Wagner <dwagner@suse.de>
---
 drivers/nvme/host/nvme.h |    5 -----
 1 file changed, 5 deletions(-)

--- a/drivers/nvme/host/nvme.h
+++ b/drivers/nvme/host/nvme.h
@@ -47,11 +47,6 @@ extern struct workqueue_struct *nvme_wq;
 extern struct workqueue_struct *nvme_reset_wq;
 extern struct workqueue_struct *nvme_delete_wq;
 
-enum {
-	NVME_NS_LBA		= 0,
-	NVME_NS_LIGHTNVM	= 1,
-};
-
 /*
  * List of workarounds for devices that required behavior not specified in
  * the standard.
